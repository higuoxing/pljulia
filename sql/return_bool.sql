CREATE FUNCTION julia_compare(x INTEGER, y INTEGER)
RETURNS BOOLEAN AS $$
    x > y
$$ LANGUAGE pljulia;
SELECT julia_compare(4,3);
DROP FUNCTION julia_compare(x INTEGER, y INTEGER);
