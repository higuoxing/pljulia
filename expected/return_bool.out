CREATE FUNCTION julia_compare(x INTEGER, y INTEGER)
RETURNS BOOLEAN AS $$
    x > y
$$ LANGUAGE pljulia;
SELECT julia_compare(4,3);
 julia_compare 
---------------
 t
(1 row)

DROP FUNCTION julia_compare(x INTEGER, y INTEGER);
